src/Core/Domain/Category/QueryResult/CategoryForTree.php

```diff
@@ -52,18 +57,21 @@ class CategoryForTree
     /**
      * @param int $categoryId
      * @param bool $active
-     * @param array<int, string> $localizedNames
-     * @param array $children
+     * @param string $name,
+     * @param string $displayName
+     * @param CategoryForTree[] $children
      */
     public function __construct(
         int $categoryId,
         bool $active,
-        array $localizedNames,
+        string $name,
+        string $displayName,
```

Comment:
Please define the BC Break

Reply:
We don't mark BC break on product page related code anymore because it was all considered experimental

Reply:
Sorry, but the class is not defined in `1.7.8.x` as `@internal` : https://github.com/PrestaShop/PrestaShop/blob/1.7.8.x/src/Core/Domain/Category/QueryResult/CategoryForTree.php

Please define BC Breaks.